### DISCUSSION TITLE
Dónde están todos?

### DISCUSSION THREAD

**Discussion by jjbagus** [atralice/Curso.Prep.Henry#55] [Q&A]
Disculpen que les escriba por acá, pero estoy muy perdido. Alguien me puede decir por favor cuál es el sitio para las consultas en general? este es el famoso SLACK o es otro lugar?
Gracias y disculpen las molestias!

**Comment by JavierArdiles** [comment]
Hola entrando a este link tenés todos los espacios de consulta. henryprep.slack.com
Te recomiendo que te descargues Slack, es mucho más cómodo.

**Reply by jjbagus** [reply]
Muchas gracias Javier! quise ingresar pero me pide usuario y contraseña y no me deja loggearme o generar un usuario. Vos recibiste alguna invitación en particular?
muchas gracias!

**Reply by JavierArdiles** [reply]
Recibí la invitación y entré así nomas. Generé el usuario y listo. Hace cuanto recibiste el mail diciendo que te iban a mandar la invitación?

**Reply by jjbagus** [reply]
Nunca me invitaron al slack... o capaz que sí y no lo ví. No se como voy a hacer ahora. Igual hoy hice el challenge... pero medio navegando en tinieblas jajajaja

**Reply by JavierArdiles** [reply]
Jajajaja a todo o nada. Fijate en Spam. A mí me llegó el mail al rato de haber mandado la entrevista y después la invitación al día siguiente.

**Reply by jjbagus** [reply]
ya fue hace rato así que estoy jugado.... ojalá lea estoy algún HenryMaster y se apiade de mí, jajaja. Muchas gracias por contestar Javier!

**Reply by carito2021z** [reply]
Yo sí encontré el mail con la invitación a Slack gracias a la respuesta de JavierArdiles porque sino no tenía ni idea dónde ir. jjbagus encontré este mail dentro de slack y tal vez te pueda servir, te deseo muchos éxitos.
 admisiones@soyhenry.com
